### DISCUSSION TITLE
Get environment Variable from Host and not from Remote

### DISCUSSION THREAD

**Discussion by NicoJolyAdeo** [module-federation/module-federation-examples#2707] [Q&A]
Hi,

In REMOTE: exposes the HelloWorld component.
In HOST: import this component.
The HelloWorld (call in host) use VUE_APP_BASE_URL_LOCK env var.
But in HOST, this env var return THE_REMOTE and not THE_HOST.

Issue : I can not use Environment variables from host in a component call from remote.

https://pbs.twimg.com/media/FnezBEXX0AEbMS2?format=jpg&name=4096x4096

Have you a fix for that ?
Thank you !

**Comment by ScriptedAlchemy** [comment]
Youre probably using DefinePlugin which converts env vars into hard coded, static values. 
Youd need some kind of runtime config that puts it on the window or as a shared module the remote can import at runtime from the host.
